mgmtsystem_security_event/model/__init__.py

```diff
@@ -0,0 +1,33 @@
+# -*- encoding: utf-8 -*-
+##############################################################################
+#
+#    OpenERP, Open Source Management Solution
+#    Copyright (C) 2015 - Present
+#    Savoir-faire Linux (<http://www.savoirfairelinux.com>).
+#
+#    This program is free software: you can redistribute it and/or modify
+#    it under the terms of the GNU Affero General Public License as
+#    published by the Free Software Foundation, either version 3 of the
+#    License, or (at your option) any later version.
+#
+#    This program is distributed in the hope that it will be useful,
+#    but WITHOUT ANY WARRANTY; without even the implied warranty of
+#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#    GNU Affero General Public License for more details.
+#
+#    You should have received a copy of the GNU Affero General Public License
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+##############################################################################
+from . import security_assets_category
+from . import security_assets_essential
+from . import security_assets_underlying
+
+from . import security_event
+from . import security_measure
+
+from . import security_threat_origin
+from . import security_threat_scenario
+
+from . import security_event_scenario
+from . import security_event_measure
```

Comment:
Alright, it's just names are getting very big. Soon it will be hard to import anything without breaking pep8.